myfirstcatapi/models/cat_model.py

```diff
@@ -169,3 +169,26 @@ def cat_summary_from_bson(cat: BSONDocument) -> dto.CatSummary:
         id=bson_id_to_cat_id(cat["_id"]),
         **cat,
     )
+
+
+async def delete_one(cat_id: dto.CatID) -> dto.UpdatedCount:
+
+    count = 0
```

Comment:
No need for this variable. Either nothing is updated or we have one updated document. 
We can return the DTO accordingly, like
```python
    except EmptyResultsFilter:
        return dto.UpdatedCount(count=0)
```
Keeping local variables always introduces some mental load ("in what cases will this variable be what value?")

WDYT?

Reply:
done